src/SPC/builder/linux/LinuxBuilder.php

```diff
@@ -293,6 +293,24 @@ protected function buildEmbed(): void
             $cwd = getcwd();
             chdir(BUILD_LIB_PATH);
             symlink($realLibName, 'libphp.so');
+            chdir(BUILD_MODULES_PATH);
+            foreach ($this->getExts() as $ext) {
+                if (!$ext->isBuildShared()) {
+                    continue;
+                }
+                $name = $ext->getName();
+                $versioned = "{$name}-{$release}.so";
+                $unversioned = "{$name}.so";
+                if (is_file(BUILD_MODULES_PATH . "/{$versioned}")) {
+                    rename(BUILD_MODULES_PATH . "/{$versioned}", BUILD_MODULES_PATH . "/{$unversioned}");
+                    shell()->cd(BUILD_MODULES_PATH)
+                        ->exec(sprintf(
+                            'patchelf --set-soname %s %s',
```

Comment:
it's a bit dirty, but I don't know a better way of dealing with this, short of patching deep into the config9.m4's of shared built-in extensions